src/indexer.cc

```diff
@@ -1611,6 +1616,36 @@ void OnIndexReference(CXClientData client_data, const CXIdxEntityRefInfo* ref) {
 FileContents::FileContents(const std::string& path, const std::string& content)
     : path(path), content(content) {}
 
+FileContentsWithOffsets::FileContentsWithOffsets() : line_offsets_{0} {}
```

Comment:
Is the {0} ctor needed?

Reply:
To be consistent. line_offsets_.size() = number of newlines + 1